src/components/profile/UserProfilePage/index.tsx

```diff
@@ -50,33 +48,26 @@ export const UserProfilePage = ({
               className={styles.profilePic}
             />
           </div>
-          <div className={styles.cardName}>
-            <Typography
-              variant="h1/bold"
-              component="h1"
-            >{`${handleUser.firstName} ${handleUser.lastName}`}</Typography>
-            <Tooltip title="Copy profile link" arrow>
-              <div className={styles.handle}>
-                <Typography
-                  variant="h5/medium"
-                  onClick={() => {
-                    copy(window.location.href);
-                    showToast('Profile link copied!');
-                  }}
-                >
-                  @{handleUser.handle}
-                </Typography>
-              </div>
-            </Tooltip>
-          </div>
-          <div className={styles.cardRank}>
-            <Typography variant="h4/regular" className={styles.rank}>
-              {getUserRank(handleUser.points)}
-            </Typography>
+          <div className={styles.userInfo}>
+            <div className={styles.cardName}>
+              <Typography variant="h1/bold" component="h1">
+                {userName}
```

Comment:
maybe name the variable `fullName` beccause when I saw `userName`, I thought that was their handle